proposals/nnnn-actors.md

```diff
@@ -120,21 +132,28 @@ Such types can then be used to annotate particular declarations that are isolate
 
 ```swift
 @UIActor
-func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) async {
   // ...
 }
 ```
 
-A declaration with an attribute indicating a global actor type is actor-isolated to that global actor. The global actor type has its own queue to protect access to the mutable state that is also actor-isolated with that same global actor type.
+A declaration with an attribute indicating a global actor type is actor-isolated to that global actor. The global actor type has its own queue that is used to perform any access to mutable state that is also actor-isolated with that same global actor.
+
+> Global actors are implicitly singletons, i.e. there is always _one_ instance of a global actor in a given process.
+> This is in contrast to `actor classes` which can have none, one or many specific instances exist at any given time. 
```

Comment:
not sure if necessary, when I first read it kind of felt like making the difference spelled out explicitly could be useful